tests/nnstreamer_plugins/unittest_plugins.cc

```diff
@@ -4342,6 +4342,109 @@ TEST (test_tensor_filter, framework_auto_ext_tflite_wrong_inputtype_n)
   g_free (test_model);
 }
 
+/**
+ * @brief Test framework auto detecion without specifying the option in tensor-filter.
+ */
+TEST (test_tensor_filter, framework_auto_wo_opt_ext_tflite)
+{
+  gchar *test_model, *str_launch_line;
+  GstElement *gstpipe;
+  const gchar fw_name[] = "tensorflow-lite";
+  GET_MODEL_PATH ("mobilenet_v1_1.0_224_quant.tflite");
+
+  str_launch_line = g_strdup_printf (
+      "videotestsrc ! videoconvert ! videoscale ! videorate ! video/x-raw,format=RGB,width=224,height=224 ! tensor_converter ! tensor_filter name=tfilter model=%s ! tensor_sink",
+      test_model);
+  gstpipe = gst_parse_launch (str_launch_line, NULL);
+  g_free (str_launch_line);
+  EXPECT_TRUE (gstpipe != nullptr);
+  TEST_TENSOR_FILTER_AUTO_OPTION_P (gstpipe, fw_name);
```

Comment:
plz add g_free (test_model);

Reply:
It Is is released in `TEST_TENSOR_FILTER_AUTO_OPTION_P` macro.
But It causes confusion by performing free resources in the macro.
Changed to release the resource here.